tempto-ldap/src/main/java/com/teradata/tempto/fulfillment/ldap/OpenLdapObjectDefinitions.java

```diff
@@ -47,9 +48,17 @@
             LdapObjectDefinition.builder("TestUser")
                     .setDistinguishedName("uid=testuser,ou=Test,dc=tempto,dc=com")
                     .setAttributes(testUserAttributes())
+                    .addModificationAttributes(memberOf())
                     .addObjectClasses(Arrays.asList("person", "inetOrgPerson"))
                             .build();
 
+    private static Map<String, List<String>> memberOf()
```

Comment:
rename to `memberOfAttributes` or `buildMemberOfAttributes`. Or just inline in TestUser definition.